packages/calcite-components/src/components/block/block.tsx

```diff
@@ -321,9 +322,7 @@ export class Block
         ) : (
           headerContent
         )}
-        {loading ? (
-          <calcite-loader inline label={messages.loading} />
-        ) : hasControl ? (
+        {hasControl && !loading ? (
```

Comment:
I _think_ `loading` doesn't affect this anymore since we want it always rendered (if a control is provided).

Reply:
Ok, it seemed like in the original code the icons rendered in the absence of loading: looks like they were built as else-if conditional ternary chains. Both for status (valid/invalid) and controls.